src/app/dataModels/project.ts

```diff
@@ -4,8 +4,18 @@ export interface Project {
   title: string;
   description: string;
   links: Link[];
-  partner?: string;
+  partners?: string;
   languages: string[];
   technologies?: string[];
   programAreas?: string;
+  location?: string;
+  status?: ProjectStatusTypes;
+  contributors: string[];
+}
+
+export enum ProjectStatusTypes {
```

Comment:
Based on our convention.

We should only have 1 class or enum per file

https://github.com/OpenBCca/openbc-web/wiki/Coding-Style-Guide

Reply:
I also think it should go in

src/app/enums/<file>